src/librustdoc/clean/types.rs

```diff
@@ -609,11 +637,26 @@ impl Attributes {
         attrs: &[ast::Attribute],
         additional_attrs: Option<(&[ast::Attribute], DefId)>,
     ) -> Attributes {
-        let mut doc_strings = vec![];
+        let mut doc_strings: Vec<DocFragment> = vec![];
```

Comment:
Hmm, why did this need to change?

Reply:
Compiler errors, only added type annotations when needed. Maybe it's not anymore, will need to be checked later on.